dev-python/email-validator/files/remove-tests-using-ns.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/tests/test_main.py b/tests/test_main.py
+index 0ed1b36..deb9154 100644
+--- a/tests/test_main.py
++++ b/tests/test_main.py
+@@ -248,23 +248,3 @@ def test_email_invalid(email_input, error_msg):
+         validate_email(email_input)
+     # print(f'({email_input!r}, {str(exc_info.value)!r}),')
+     assert str(exc_info.value) == error_msg
+-
+-
+-def test_deliverability_no_records():
```

Comment:
We generally prefer `sed -i -e 's:test_foo:_&:'` kind of thing to disable tests. Causes less maintenance effort in the future.

Reply:
Also please file an upstream bug explaining why relying on internet is bad, and that this should be easily avoided by mocking.

Reply:
https://github.com/JoshData/python-email-validator/issues/42